packages/nimble-components/tsconfig.json

```diff
@@ -19,6 +19,7 @@
         // Require type only imports where possible
         // so imports with side-effects are explicit
         "importsNotUsedAsValues": "error",
```

Comment:
Do you know if there's a reason we set `"importsNotUsedAsValues": "error"` in only two of our packages? Needn't be part of this PR, but should we consider enabling it everywhere?

Reply:
It's most important in our tsconfigs that have imports with side-effects, i.e. anything that imports directly from nimble-components using the side-effect syntax, i.e. `import '@ni/nimble-components/dist/esm/button';`. So site and performance packages should probably have it to be safe.

Doesn't hurt to be consistent, I'll look at consistency across our tsconfigs. May be worth an internal private package to share that tsconfig inside nimble, could eventually lead to re-opening: https://github.com/ni/javascript-styleguide/pull/53

Think I'll put it on my personal list to play with. But if worth tracking as tecch debt lemme know and I can put that together.